config/locales/de.yml

```diff
@@ -0,0 +1,204 @@
+de:
+  title: "RubyGems.org"
+  subtitle: "Deine Community des Gem-Hostingservices"
+  help_url: "http://help.rubygems.org"
+  please_sign_up: "Zugriff verweigert. Bitte melden Sie in https://rubygems.org mit einem Konto an"
+  feed_subscribed: "RubyGems.org | Abonnierte Gems"
+  feed_latest: "RubyGems.org | Neueste Gems"
+  time_ago: "seit %{duration}"
+  form_disable_with: "Bitte warten..."
+  update: Aktualisieren
+  edit: Bearbeiten
+  sign_in: "Anmelden"
+  sign_up: "Registrieren"
+  locale_name: "German"
+  this_rubygem_could_not_be_found: "Dieser Ruby Gem konnte nicht gefunden werden."
+
+  dashboards:
+    show:
+      latest: "Letzte Aktualisierungen"
+      latest_title: "Neueste Aktualisierungen RSS Feed"
+      no_subscriptions: "Du hast noch keine Gems abonniert. Besuche %{gem_link} um ein Gem zu abonnieren!"
```

Comment:
here ("ein Gem") you are using "das Gem" (otherwise it would be "einen Gem"). Should be consistent (see above).